### PR TITLE
Add disable-snapshot-purge setting information

### PR DESCRIPTION
**Pull request by ejweber**
longhorn/longhorn#7075

Keep as a draft until backports are complete (may need to update multiple versions).

### CODE REVIEW

**Review by innobead — APPROVED** [review]

File: content/docs/1.4.5/references/settings.md [hunk 1]
```diff
@@ -647,6 +648,18 @@ Notice that trying to trim a removed files from a valid snapshot will do nothing
...
+
+> Default: `false`
+
+When set to true, temporarily prevent all attempts to purge volume snapshots.
+
+Longhorn typically purges snapshots during replica rebuilding and user-initiated snapshot deletion. During this process,
+Longhorn coalesces unnecessary snapshots into their newer counterparts, freeing space consumed by historical data. It
+is always desirable to allow snapshot purging during normal operation. However, purging does temporarily consume
+additional disk space. If an underlying disk is too full for a snapshot purge operation to complete successfully, it may
+be beneficial to temporarily disable purging while data is moved to other disks.
```
Comment (jillian-maroket) [inline content/docs/1.4.5/references/settings.md:661]:
```suggestion
Longhorn typically purges snapshots during replica rebuilding and user-initiated snapshot deletion. While purging, Longhorn coalesces unnecessary snapshots into their newer counterparts, freeing space consumed by historical data.

Allowing snapshot purging during normal operations is ideal, but this process temporarily consumes additional disk space. If insufficient disk space prevents the process from continuing, consider temporarily disabling purging while data is moved to other disks.
```

Comment (jillian-maroket) [reply content/docs/1.4.5/references/settings.md:661]: I suggest having two paragraphs so users can scan the text more easily. The first one explains concepts while the second one provides recommendations. I also reworded some parts to reduce the word count.

**Review by jillian-maroket — COMMENTED** [review]
@ejweber Let me know if you have concerns about the changes.

File: content/docs/1.4.5/references/settings.md (hunk 1 above)
Comment (ejweber) [reply content/docs/1.4.5/references/settings.md]: No concerns. Fixed by amending the commit.

**Review by jillian-maroket — APPROVED** [review]

**Review by innobead — APPROVED** [review]

**Pull request merged**